packages/rancher-monitoring/generated-changes/patch/values.yaml.patch

```diff
@@ -294,8 +294,8 @@
 +    ## Every Windows host must have a wins version of 0.1.0+ to use this chart (default as of Rancher 2.5.8).
 +    ## To upgrade wins versions on Windows hosts, see https://github.com/rancher/wins/tree/master/charts/rancher-wins-upgrader.
 +    ##
-+    # windows:
-+    #   enabled: false
++    windows:
++      enabled: false
```

Comment:
Uncommented this since leaving it commented out caused windowsExporter to be deployed by default.